docs/fargate.md

```diff
@@ -115,6 +120,135 @@ fi
 
 `EFS_ID` is going to be used in setting up Volumes for metric pods.
 
+## Logs
+
+The following are some of the steps needed to setup and enable logs collection on Fargate
+
+### Fluent-bit log router configuration
+
+Below are the steps to configure the log router on fargate
+
+- Create a dedicated Kubernetes namespace named `aws-observability`
+
+```yaml
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: aws-observability
+  labels:
+    aws-observability: enabled
+```
+
+The fargate log router manages the `Service` and `Input` sections. One needs to create a `ConfigMap` to enable log routing to cloudwatch, an
+example is shown below:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: aws-logging
+  namespace: aws-observability
+data:
+  flb_log_cw: "true" # Set to true to ship Fluent Bit process logs to CloudWatch.
+  output.conf: |
+    [OUTPUT]
+        Name cloudwatch_logs
+        Match   kube.*
+        region region-code
+        log_group_name my-logs
+        log_stream_prefix from-fluent-bit-
+        log_retention_days 60
+        auto_create_group true
+```
+
+For more information on this refer to [fargate-logging](https://docs.aws.amazon.com/eks/latest/userguide/fargate-logging.html)
+
+### Cloudwatch logs collection
+
+After setting up fluent-bit to forward logs to cloudwatch, the next step is to setup and enable cloudwatch collection
+
+This involves the following:
+
+- [Setup Cloudwatch authentication](#authenticate-with-cloudwatch)
+- [Enable Clouwatch collection](#enable-cloudwatch-collection-eks-fargate-only)
+
+#### Authenticate with Cloudwatch
+
+To configure the service account to use an IAM role (for authentication), follow the steps below
+
+- Set your AWS account ID to an environment variable
+- Set your cluster's OIDC identity provider to an environment variable. Replace `my-cluster` with the name of your cluster
+- Set the namespace and the service account name
+
+```bash
+account_id=$(aws sts get-caller-identity --query "Account" --output text)
+oidc_provider=$(aws eks describe-cluster --name my-cluster --region $AWS_REGION --query "cluster.identity.oidc.issuer" --output text | sed -e "s/^https:\/\///")
+export namespace=default
+export service_account=my-service-account
```

Comment:
Please use upper case variable names, also is this `namespace` the same like `NAMESPACE` mentioned on the top of this document? We should be consistent regarding the variables within the document

Reply:
yes its the same namespace so, I'll remove this reference